File: Array/T500_Keyboard Row/PYTHON/main.py
from typing import List
class Solution:
    def findWords(self, words: List[str]) -> List[str]:
        ans = []
        t = "12210111011122000010020202"
        for word in words:
            idx = t[ord(word[0].lower()) - ord('a')]
            if all(t[ord(c.lower()) - ord('a')] == idx  for c in word):
                ans.append(word)
        return ans

File: Array/T500_Keyboard Row/GO/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	words := []string{"Hello", "Alaska", "Dad", "Peace"}
	fmt.Println("case1:", findWords(words))
}

func findWords(words []string) []string {
	ans := []string{}
	t := "12210111011122000010020202"
	for _, v := range words {
		s := strings.ToLower(v)
		sLen := len(s)
		rowindex := t[s[0]-'a']
		i := 0
		for ; i < sLen; i++ {
			if t[s[i]-'a'] != rowindex {
				break
			}
		}

		if i == sLen {
			ans = append(ans, v)
		}
	}

	return ans
}
